README.rst

```diff
@@ -390,6 +390,13 @@ Choose from the list of available rules:
 
   Calling ``unset`` on multiple items should be done in one call.
 
+* **combine_dirname** [@PHP70Migration:risky, @PHP71Migration:risky]
```

Comment:
combine dirname with... ?
`combine_multiple_dirname` or `combine_nested_dirname` ?

Reply:
+1 for `combine_nested_dirname`